python_examples/simulation_outputs/output.csv

```diff
@@ -1,61 +1 @@
 time,InfectionStatus.Susceptible,InfectionStatus.Exposed,InfectionStatus.InfectASympt,InfectionStatus.InfectMild,InfectionStatus.InfectGP,InfectionStatus.InfectHosp,InfectionStatus.InfectICU,InfectionStatus.InfectICURecov,InfectionStatus.Recovered,InfectionStatus.Dead
-0,95,0,0,5,0,0,0,0,0,0
```

Comment:
Why has this been overwritten? Unless there is a specific reason, it should be left as is, and that change discarded.

Reply:
Similarly, the output plot should _not_ have been deleted.

Reply:
I think this is changes made by @Elizabeth-Hayman . I only touched the household ages sweep and corresponding test I'm not sure what this is.